netboot/debian-installer/i386/boot-screens/menu.cfg

```diff
@@ -0,0 +1,12 @@
+menu hshift 7
+menu width 61
+
+menu title PXE server boot menu
+label install
+	menu label ^Install
+	kernel debian-installer/i386/linux
+	append initrd=debian-installer/i386/initrd.gz --- console=ttyS0,115200 earlyprint=serial,ttyS0,115200
+label run
+	menu label ^RunNFS
+	kernel linux-4.8.5
+	append root=/dev/nfs rw ip=dhcp nfsroot=192.168.0.109:/home/miczyg/nfs,vers=4,tcp --- console=ttyS0,115200 earlyprint=serial,ttyS0,115200
```

Comment:
I could not set up nfs in docker, because of rpcbind problems. Have not found solution yet.